iocage_lib/ioc_create.py

```diff
@@ -611,11 +611,15 @@ def _create_jail(self, jail_uuid, location):
                 silent=self.silent)
 
         if self.pkglist:
-            if config.get('ip4_addr', 'none') == 'none' and \
-                config.get('ip6_addr', 'none') == 'none' and \
-                    not iocage_lib.ioc_common.check_truthy(
-                        config.get('dhcp', 0)
-            ):
+            dhcp_or_hostname = iocage_lib.ioc_common.check_truthy(
+                config.get('dhcp', 0)
```

Comment:
I might be wrong here, but umm isn't the config supposed to have `0` or `1` ?

Reply:
It *may* not exist as a key at this point, so using get for safety.

Reply:
But if it does, it will be `0` or `1` ?

Reply:
Right, so if the key doesn't exist we set 0 as a safety net.

Reply:
Perfect. So if it exists, i don't think we need use check_truthy in that case ? We can safely perhaps do `if config.get(key, default)` ?

Reply:
The user may be supplying this property and it could be a variation of the bools, that's where `check_truthy` comes in

Reply:
Ah I see. I thought we maybe had parsed config at this point and it contained `0` or `1` only for boolean based keys. So the config is normalized after this then ?